packages/orchestration/test/staking-ops.test.ts

```diff
@@ -174,17 +178,16 @@ const makeScenario = () => {
     return { zcf, zoe };
   };
 
-  const makeRecorderKit = () => harden({}) as any;
-
   const baggage = makeScalarBigMapStore('b1') as Baggage;
   const zone = makeDurableZone(baggage);
+  const marshaller = makeFakeBoard().getReadonlyMarshaller();
+  const { makeRecorderKit } = prepareRecorderKitMakers(baggage, marshaller);
 
   const { delegations, startTime } = configStaking;
 
-  // TODO: when we write to chainStorage, test it.
-  //   const { rootNode } = makeFakeStorageKit('mockChainStorageRoot');
-
-  const storageNode = Far('StorageNode', {}) as unknown as StorageNode;
+  const { rootNode } = makeFakeStorageKit('mockChainStorageRoot', {
```

Comment:
consider a unique value for easier string search
```suggestion
  const { rootNode } = makeFakeStorageKit('stakingOpsTest', {
```